elcid/models.py

```diff
@@ -758,6 +782,47 @@ def get_api_name(cls):
         return camelcase_to_underscore(cls._meta.object_name)
 
 
+class ChronicAntifungal(omodels.PatientSubrecord):
+    DRUG_LIST = "drug_list"
+    REASON_TO_INTERACTION = "Reason for interaction"
+
+    REASONS = (
+        (DRUG_LIST, DRUG_LIST,),
+        (REASON_TO_INTERACTION, REASON_TO_INTERACTION,),
+    )
+    updated_dt = models.DateTimeField(auto_now=True)
+    reason = models.TextField(
+        choices=REASONS, blank=True, null=True
+    )
+
+    ACTIVE_DAYS = 3
+
+    @classmethod
+    def active_from_date(cls):
+        return timezone.now().date() - datetime.timedelta(
+            cls.ACTIVE_DAYS
+        )
+
+    @property
+    def active(self):
+        active_from_date = self.__class__.active_from_date()
+        days_ago = datetime.date.today() - datetime.timedelta(
```

Comment:
Is this right ? Why would this be today and then compared to a classmethod that looks at now() ?